frontend/packages/topology/src/filters/TopologyFilterBar.scss

```
@@ -1,12 +1,14 @@
 @import '../../../../public/style/vars';
 
+
+
 .odc-topology-filter-bar {
   background-color: var(--pf-global--BackgroundColor--100);
-  padding: 6px $pf-global-gutter;
+  padding: 6px 0rem;
 
-  @media (min-width: $pf-global--breakpoint--xl) {
-    padding-left: $pf-global-gutter--md;
-    padding-right: $pf-global-gutter--md;
+  @media screen and (min-width: $pf-global--breakpoint--xl){
```

The diff shows here that you removed the space here, but our other code has it, so please keep it :)

```suggestion
  @media screen and (min-width: $pf-global--breakpoint--xl) {
```